src/components/controls/Controls.tsx

```diff
@@ -54,54 +55,63 @@ export const Controls: FC<ControlProps> = ({
 	})();
 
 	// useEventListener uses under the hood event listeners, that are also present in api, but do not correspond typings from the package
+	// Play animation on `play` event
 	useEventListener(
 		'play',
-		() => setHasStarted(true),
-		api as unknown as HTMLElement,
-	);
-
-	// Play animation when video is paused
-	useEventListener(
-		'pause',
 		() => {
+			setHasStarted(true);
 			if (!hasStarted) {
 				return;
 			}
```

Comment:
why this:
```ts
setHasStarted(true);
  if (!hasStarted) {
	return;
 }
```
You set it to `true` and then check if it isn't `true`

Reply:
The logic is a lil bit cumbersome, but it check if video was played before (we do not run play animation on “pre-play" screen— when we have big play button ).
And for that we have a getter `api.getHasPlayedOrSeeked()`, so keeping that state is not valid .
replacing